index.html

```diff
@@ -0,0 +1,70 @@
+
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="style.css"/>
+        <title>Odin Landing Page</title>
+        <meta charset="utf-8">
+    </head>
+
+    <body>
+        <div class="top">
+            <div class="header">
+                <p class="logo">Header Logo</p>
+                <div class="links">
+                    <a href="#">header link one</a>
+                    <a href="#">header link two</a>
+                    <a href="#">header link three</a>
+                </div>
+            </div>
+            <div class="first">
+                <div class="first-container">
+                    <h1 class="first-header">This website is awesome</h1>
+                    <p class="first-header-subtext">This website has some subtext that goes under the main title. It's a smaller font and the color is lower contrast</p>
+                    <button class="first-button">Sign up</button>
+                </div>
+                <div class="placeholder">
+                <p >this is a placeholder for an image</p>
+                </div>
+            </div>
+        </div>
+        <div class="second">
+            <h1 class="second-header">Some random information</h1>
+            <div class="images-second">
+                <div class="image-subtext">
+                    <img src="images/image1.jpg">
+                    <p>this is some subtext under an illustration or image</p>
+                </div>
+                <div class="image-subtext">
+                    <img src="images/image2.jpg">
+                    <p>this is some subtext under an illustration or image</p>
+                </div>
+                <div class="image-subtext">
+                    <img src="images/image3.jpg">
+                    <p>this is some subtext under an illustration or image</p>
+                </div>
+                <div class="image-subtext">
+                    <img src="images/image1.jpg">
+                    <p>this is some subtext under an illustration or image</p>
+                </div>
+            </div>
+        </div>
+
+        <div class="third">
```

Comment:
consider using section tags for emphasizing blocks,

for this particular section - a figure & a blockquote tag can also be considered